### ISSUE TITLE
Does not load folders which have large number of files

### ISSUE THREAD

**Issue by sawsrinath** [ipfs/ipfs-webui#1907]
I have uploaded a large number of files(11GB of images) from ipfs desktop app. It went fine until I have uploaded 14000 images. But after that, the app keeps loading and returning to the main folder. All the files are on my pc. A small number of files uploaded fine and can be viewed inside the folder

![Capture](https://user-images.githubusercontent.com/99781744/154195058-1a9e9e79-9ad2-4d59-a8ca-3a1810cb69e9.JPG)

**Comment by lidel** [comment]
@sawsrinath

- Were those files (14000  of them) all in a single directory? 
- Are you able to do some CLI debugging? 
   - does `ipfs files ls /path/to/your/big/dir` work, or does it hang too?
   - what is the number returned by `ipfs files ls /path/to/your/big/dir | wc -l`

**Comment by sawsrinath** [comment]
-yes
- i cannot type anything on CLI

![Capture](https://user-images.githubusercontent.com/99781744/156484331-6cc27f2f-9377-4764-8097-8b2be2100caf.JPG)

**Comment by lidel** [comment]
@sawsrinath you should be able to start `ipfs daemon` in one terminal and then run `ipfs files` commands i listed in another one.

**Comment by beauwilliams** [comment]
Can we reopen. I've tried everything and it seems there is no way to upload a large folder (70k files) reliably.

Here is the CLI output
```
ls /Users/admin/Git_Downloads/AfricaRare/AfricarareLandDataParser/output/metadata | wc -l
   71504
```

Things I have tried
- IPFS Desktop importing folder (hangs saying importing 1 item at 0%, or crashes the UI)
- Pinata CLI, errors at end after uploading all files (it works intermittently even with error, but now not at all)
- NFT UP app [mac m1 version] (hangs at 71,502 files, 0.01B of 20.75MB)
- Pinata, NFT.Storage and more websites (they all crash)

Is there a recommended and reliable way to upload a large # of files in a folder? I'm all out of options here it seems.

 I managed to upload 80k file folders in past, but only ever got the pinataCLI to work. Now I am at a dead end it seems. Appreciate the help!
